ansible/devutils

```diff
@@ -348,6 +372,9 @@ def main():
                         type=str, required=False, default='lab')
     parser.add_argument('-l', '--limit', help='Host: limit to a single dut host name, default all',
                         type=str, required=False)
+    parser.add_argument('--log-level', help='Log level: print logs to STDOUT (if not set to close)', 
+                        type=str, required=False, default='close',
```

Comment:
@Blueve I choose `close` as the default value because this is the same behavior as current code. Actually, I think `warn` is a better one because we can get the `WARNING` and `ERROR` logs by default. How do you think?

Reply:
I think we can print WARNING/ERROR by default.

Reply:
Thanks! I updated it in my latest commit.